google/cloud/security/common/data_access/csv_writer.py

```diff
@@ -329,6 +328,7 @@
     'raw_instance_group_manager',
 ]
 
+
```

Comment:
I realize that we have not been consistent in this particular file, but could you remove this newline?